components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/test/java/org/wso2/carbon/identity/notification/sender/tenant/config/NotificationSenderManagementServiceImplTest.java

```diff
@@ -57,15 +57,17 @@ public class NotificationSenderManagementServiceImplTest {
     private ChannelConfigurationHandler websubhubChannelConfigurationHandler;
     @Mock
     private ConfigurationManager configurationManager;
+    private String defaultHandlerName = "default";
```

Comment:
make these constants, if we have the `default` already set as public constant, we can reuse it here.